src/DIRAC/Resources/Computing/AREXComputingElement.py

```diff
@@ -39,335 +29,257 @@
-        "Killed": "Killed",
-        "Failed": "Failed",
-        "Hold": "Failed",
-        "Finished": "Done",
-        "Done": "Done",
-        "Other": "Done",
-    }
-
-    #############################################################################
     def __init__(self, ceUniqueID):
         """Standard constructor."""
         super(AREXComputingElement, self).__init__(ceUniqueID)
-        self.ceType = CE_NAME
-        self.submittedJobs = 0
-        self.mandatoryParameters = MANDATORY_PARAMETERS
-        self.pilotProxy = ""
-        self.queue = ""
-        self.outputURL = "gsiftp://localhost"
-        self.ceHost = self.ceName
-
-        result = returnValueOrRaise(getCESiteMapping(self.ceHost))
-        self.site = result[self.ceHost]
-        self.grid = self.site.split(".")[0]
+
+        # Default REST port
+        self.port = "443"
+        # REST version to adopt
+        self.restVersion = "1.0"
+        # Time left before proxy renewal: 2 hours, 46 minutes and 40 seconds
+        self.proxyTimeLeftBeforeRenewal = 10000
+        # Timeout
+        self.arcRESTTimeout = 5.0
+        # Request session
+        self.session = None
+        self.headers = {}
+        # URL used to communicate with the REST interface
+        self.base_url = ""
 
     #############################################################################
-    @property
-    def initialised(self):
-        """Check if the REST interface is available and ready, configuring it if required.
-        Hopefully this is executed just once per CE, though it should not harm to run it more often.
-        If all is well the following variables are set, and a few others on which they (especially self.s) depend.
-        self.initialised : Static variable to say if the REST interface is available and ready
-        self.base_url: The basic URL for the REST interface
-        self.s       : The python "requests" module session
-        self.headers : The format of information we prefer : json of course
-        Specification : https://www.nordugrid.org/arc/arc6/tech/rest/rest.html
 
-        The following needed variables are obtained from the CS. If not available, some hopefully
-        sensible defaults are set.
-        "RESTEndpoint"      - CE is Queried if not available in CS
-               - The endpoint we talk to
-        "XRSLExtraString" - Default = ""
-               - Any CE specific string with additional parameters
-        "XRSLMPExtraString" - Default = ""
-               - Any CE specific string with additional parameters for MP jobs
-        "ARCRESTTimeout"    - DEfault = 1.0 (seconds)
-               - Timeout for the rest query
-        "proxyTimeLeftBeforeRenewal" - Default = 10000 (seconds)
-               - As the name says
-
-        Note : This is not run from __init__ as the design of DIRAC means that ceParameters is
-        as yet only filled with CEDefaults at the time this class is initialised for the given CE
+    def _reset(self):
```

Comment:
Do we really want to overload the _reset method? Isn't t simpler to retain the original name for this method?

Reply:
Yes, the `_reset()` method is used by the `ComputingElement` class to initialize any CE.
It is the "standard" way of getting parameters from the `ceParameters` dictionary.